src/htdocs/partials/sensors/table.php

```diff
@@ -112,7 +112,17 @@
         <?php echo implode(' | ', $weather) ?>
         <br/>
         <?php endif ?>
-        <small><?php echo __('Last update') ?>: <?php echo date("Y-m-d H:i:s", $sensors['last_update']); ?>.</small>
+        <small
+          <?php if ((time() - $sensors['last_update']) > 15 * 60): ?>
+            <?php echo "style='color: #ff1f1f'" ?>
```

Comment:
Let's use a proper CSS class instead of inline styles. The class name can be computed before `<small>`, eg.:
```php
<?php
if (...) {
  $lastUpdatedClass="...";
} else if (...) {
  $lastUpdatedClass="...";
} else {
  $lastUpdatedClass="";
}
?>
<small class="<?php echo $lastUpdatedClass ?>">
```

Reply:
No problemo. Where do you keep such css styles? Could you provide the file path?

Reply:
Is it `src/htdocs/public/css/style.css`?

Reply:
Yes.